src/DynamoCoreWpf/Views/PackageManager/TermsOfUseView.xaml

```diff
@@ -0,0 +1,69 @@
+﻿<Window x:Class="Dynamo.PackageManager.UI.TermsOfUseView"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+        xmlns:ui="clr-namespace:Dynamo.UI"
+        xmlns:p="clr-namespace:Dynamo.Wpf.Properties"
+        xmlns:views="clr-namespace:Dynamo.UI.Views"
+        Height="450"
+        Width="550"
+        Background="#555"
+        MinWidth="550"
+        MaxWidth="550"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:ViewModels="clr-namespace:Dynamo.ViewModels"
+        mc:Ignorable="d"
+        Title="{x:Static p:Resources.TermsOfUseViewTitle}"
+        d:DataContext="{d:DesignInstance ViewModels:DynamoViewModel, IsDesignTimeCreatable=False}">
+
+    <Window.Resources>
+        <ResourceDictionary>
+            <ResourceDictionary.MergedDictionaries>
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoModernDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoConvertersDictionaryUri}" />
+                <ui:SharedResourceDictionary Source="{x:Static ui:SharedDictionaryManager.DynamoColorsAndBrushesDictionaryUri}" />
```

Comment:
Can you see which of these `SharedResourceDictionary` is not needed, then remove it?